taca/analysis/analysis_element.py

```diff
@@ -92,15 +93,25 @@ def _process(run):
 
         elif demultiplexing_status != "finished":
             logger.warning(
-                f"Unknown demultiplexing status {demultiplexing_status} of run {run}. Please investigate."
+                f"Unknown demultiplexing status {demultiplexing_status} of run {run}. "
+                "Please investigate."
             )
             email_subject = f"Issues processing {run}"
-            email_message = f"Unknown demultiplexing status {demultiplexing_status} of run {run}. Please investigate."
+            email_message = (
+                f"Unknown demultiplexing status {demultiplexing_status} of "
+                f"run {run}. Please investigate."
+            )
             send_mail(email_subject, email_message, CONFIG["mail"]["recipients"])
             return
 
         email_subject = f"Demultiplexing completed for {run}"
-        email_message = f"Demultiplexing completed without errors for {run}. Starting transfer to analysis cluster"
+        email_message = f"""The run {run} has been demultiplexed without any error or warning.
+
+                    The Run will be transferred to the analysis cluster for further analysis.
+
+                    The run is available at https://genomics-status.scilifelab.se/flowcells_element/{run}
+                    
+                    """
```

Comment:
Won't an indented multi-line string like this cause the email contents to also be indented? 🤔 I've been thinking about this before and we might be able to fix it like this.

```suggestion
        from textwrap import dedent # Put this in the top of the file
        email_message = dedent(f"""\
            The run {run} has been demultiplexed without any error or warning.
            The Run will be transferred to the analysis cluster for further analysis.
            The run is available at https://genomics-status.scilifelab.se/flowcells_element/{run}
            """)
                
```

Reply:
Hm, I never thought of that 😅 I think I prefer to just split the string and slap on some `\n`s  though.